src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MaxLatencySpec.groovy

```diff
@@ -201,34 +206,33 @@ class MaxLatencySpec extends HealthCheckSpecification {
 
         then: "Flow is rerouted and goes to the DEGRADED state"
         wait(rerouteDelay + WAIT_OFFSET) {
-            def flowHistory = flowHelper.getLatestHistoryEntry(flow.flowId)
-            flowHistory.payload.last().action == REROUTE_SUCCESS
+            def flowLastHistoryEntry = flow.retrieveFlowHistory()
+                    .getEntriesByType(FlowActionType.REROUTE).last()
+            flowLastHistoryEntry.toString() == REROUTE_SUCCESS
```

Comment:
Could you please use FlowActionType instead of FlowHistoryConstants (REROUTE_SUCCESS -> REROUTE)

Reply:
Fixed